### PR TITLE
Update COS references to cos-beta family

### PR DESCRIPTION
**Pull request by ndixita**

### CODE REVIEW

File: jobs/e2e_node/containerd/k8s-release-branches/image-config-1.25.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 images:
   cos-101:
-    image_family: cos-101-lts
+    image_family: cos-beta
```
Comment (mmiranda96) [inline jobs/e2e_node/containerd/k8s-release-branches/image-config-1.25.yaml:3]: Don't update this one, 1.25 is mapped to a fixed family.

File: jobs/e2e_node/containerd/k8s-release-branches/image-config-1.26.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 images:
   cos-101:
-    image_family: cos-101-lts
+    image_family: cos-beta
```
Comment (mmiranda96) [inline jobs/e2e_node/containerd/k8s-release-branches/image-config-1.26.yaml:3]: Same here.

File: jobs/e2e_node/containerd/k8s-release-branches/image-config-1.27.yaml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 images:
   cos-105:
-    image_family: cos-105-lts
+    image_family: cos-beta
```
Comment (mmiranda96) [inline jobs/e2e_node/containerd/k8s-release-branches/image-config-1.27.yaml:3]: Same here.

**Review by mmiranda96 — COMMENTED** [review]
Some minor comments.

**Review by mmiranda96 — APPROVED** [review]
/lgtm

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

do you know when it will go to stable?

**Comment by mmiranda96** [comment]
@SergeyKanzhelev COS 109 is expected to reach LTS by the end of September 2023.

**Pull request merged**